local/network.go

```diff
@@ -447,6 +448,7 @@ func (ln *localNetwork) Healthy(ctx context.Context) chan error {
 	ln.lock.RLock()
 	defer ln.lock.RUnlock()
 
+	zap.L().Info("checking local network healthiness", zap.Int("nodes", len(ln.nodes)))
```

Comment:
is this integrated with the logging object?

Reply:
Yes, the network runner RPC server will log these to stderr